packages/react-icon-provider/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
```

Comment:
Just remembered, you should delete this file (we removed it from the template and from existing packages after the PR was created)